services/galley/src/Galley/Cassandra/Instances.hs

```diff
@@ -200,13 +199,14 @@ instance Cql GroupId where
   fromCql (CqlBlob b) = Right . GroupId . LBS.toStrict $ b
   fromCql _ = Left "group_id: blob expected"
 
-instance Cql GroupInfoBundle where
+instance Cql (RawMLS PublicGroupStateTBS) where
   ctype = Tagged BlobColumn
 
-  toCql = CqlBlob . runPut . serialiseMLS
-
-  fromCql (CqlBlob b) = mapLeft T.unpack $ decodeMLS @GroupInfoBundle b
-  fromCql _ = Left "GroupInfoBundle: blob expected"
+  toCql = CqlBlob . LBS.fromStrict . rmRaw
+  fromCql (CqlBlob b) =
+    mapLeft T.unpack $
+      decodeMLS @(RawMLS PublicGroupStateTBS) b
+  fromCql _ = Left "PublicGroupStateTBS: blob expected"
```

Comment:
@pcapriotti , does this meet your expectations? The part we cannot avoid (if we want to have the `RawMLS PublicGroupStateTBS` type here) is decoding, can we? An alternative is to have a `ByteString` (or a newtype around it) instead that would hold just the raw bytes, but then I don't know how this will play out once we start supporting the delta and by-reference kinds of public group states.

Reply:
I'd rather just read a bytestring from the database. For the moment, we don't need to parse it, and if we eventually do, we're probably not going to serialise it as a blob anyway (but I'm not sure).

Also, why is it `PublicGroupStateTBS`? Shouldn't it be the whole `PublicGroupState`?

Reply:
Ok, the bytestring variant sounds reasonable. I'll do that.

Regarding `PublicGroupStateTBS` vs. `PublicGroupState`, I've put the TBS one assuming that for these purposes we don't need the signature of the state. As I type this, it sounds like it'd be useful to have a signature as well, wouldn't it? I can put it for there. Should I use `PublicGroupState` instead?

Reply:
How were you planning to reconstruct the signature when serving it to the clients?